### ISSUE TITLE
Clean up interface and data structures for V1.0

### ISSUE THREAD

**Issue by Zulu-Inuoe** [Zulu-Inuoe/pjlink#4]
I'm not super happy with the data structures and interface right now.

### projector-input 
Can just encode as `(type . number)` conses. Less type info but it's easier to work with.
### projector-status
For this it makes sense to encode it as an alist or plist, and rather than `:ok`, use `nil` to signify 'no error':

example:
```lisp
(:fan nil
 :lamp nil
 :temperature nil
 :cover-open :error
 :filter :warning
 :other nil)
```
### projector-lamp
it might also make sense to not bother recording the number and also turn them into `(hours . status)` conses. The lamp number would be implicit in the list.

### projector-resolution
Ditto on simple `(horizontal . vertical)` resolution

## Interface
Basically I'm not sure if the extra trouble I'm going through to support `pjlink-config` is worthwhile.
The point of having the extra method is to save users from having to pass all the necessary keyword args (primarily `:password`) repeatedly.
In the future I want to implement some sort of 'keep-alive' connection for cases where many commands are being done in a short time, so that could be enough for the case where the user would have to specify all the &key params.
It would also be better for security purposes since the connection need only remember the connection digest, not the actual password. And even that could be thrown away after first command.

**Comment by Zulu-Inuoe** [comment]
Changed out classes for conses and alists in https://github.com/Zulu-Inuoe/pjlink/commit/f1ac11401144d3243f56121986e201e810c4d1fd

I'll see how I like it for now.
As per the persistent connection, I'll leave it as it is for now and tackle it in some future version
